src/app/components/pages/IsaacCompetition/IsaacCompetition.tsx

```diff
@@ -89,6 +91,67 @@ export const IsaacCompetition = () => {
           </Container>
         </div>
       </section>
+      <section id="competition-information-section">
+        <Container className="pt-4 pb-4 z1">
+          <Row className="py-4">
+            <Col xs={12} lg={6}>
+              <InformationCard
+                title={section3.howItWorks.title}
+                content={section3.howItWorks.steps}
+                className="competition-information-default-background"
+              />
+            </Col>
+            <Col xs={12} lg={6} className="mt-4 mt-lg-0">
+              <InformationCard
+                title={section3.whyJoin.title}
+                description={section3.whyJoin.description}
+                content={section3.whyJoin.benefits}
+                className="competition-information-default-background"
+              />
+            </Col>
+          </Row>
+          <Row className="py-4">
+            <Col xs={12} lg={6}>
+              <InformationCard
+                title={section3.eligibility.title}
+                content={[section3.eligibility.description, section3.eligibility.requirements]}
+                className="competition-information-default-background"
+              />
+            </Col>
+            <Col xs={12} lg={6} className="mt-4 mt-lg-0">
+              <InformationCard
+                title={section3.prizes.title}
+                description={section3.prizes.description}
+                content={section3.prizes.prizeList}
+                isList
+                className="competition-information-prizes-background"
+              />
+            </Col>
+          </Row>
+          <section className="competition-timeline">
+            <h3 className="competition-timeline-title">{section3.timeline.title}</h3>
+            <p className="competition-timeline-content">{section3.timeline.content}</p>
+            <Row className="p-4">
+              {section3.timeline.entries.map((entry, index) => (
+                <Col
```

Comment:
How come this isn't part of the component with the arrow? It makes complete sense for this to be within `<CompetitionTimeline/>`, especially the arrow :)